python/pyarrow/plasma.pyx

```diff
@@ -248,12 +250,7 @@ cdef class PlasmaClient:
             check_status(self.client.get().Get(ids.data(), ids.size(),
```

Comment:
Should this be changed to use `GetAuto`?

Reply:
Yes, I had overlooked that. Thank you.